src/includes/classes/Plugin.php

```diff
@@ -345,6 +346,7 @@ public function setup()
             'htmlc_compress_css_code'              => '1', // `0|1`.
             'htmlc_compress_js_code'               => '1', // `0|1`.
             'htmlc_compress_html_code'             => '1', // `0|1`.
+            'htmlc_when_logged_in' => '0', // `0|1`; enable when logged in?
```

Comment:
@renzms I noticed the formatting is slightly off here; notice how your addition here doesn't line up with the options above it.